src/bin/add_historical_data.rs

```diff
@@ -0,0 +1,527 @@
+        let p2_result = parse_race_result(p2_time).unwrap();
+        let outcome: Outcome = From::from((&p1_result, &p2_result));
+        Self {
+            player_1_name: p1.to_string(),
+            player_2_name: p2.to_string(),
+            player_1_result: p1_result,
+            player_2_result: p2_result,
+            outcome,
+        }
+    }
+}
+
+struct MakeBracket {
+    new_bracket_name: &'static str,
+    backfill_note: String,
+    races: Vec<Vec<RaceResult>>,
+    bracket_type: BracketType,
+}
+
+struct MakeSeason {
+    new_season: NewSeason,
+    brackets: Vec<MakeBracket>,
+}
+
+// TODO: standings not populating - probably need to set bracket state or something?
+fn main() -> anyhow::Result<()> {
+    dotenv::dotenv()?;
+    let no_times_backfill_note = "This is a backfill of a historical bracket that was run off-site. The times included here are placeholders. They may be updated at some future time.".to_string();
+    let dark_world = MakeBracket {
+        new_bracket_name: "Dark World",
+        backfill_note: no_times_backfill_note.clone(),
+        bracket_type: BracketType::Swiss,
+        races: vec![
+            vec![
+                ("foxlisk", "buane"),
+                ("eriror", "mooglemod"),
+                ("doomtap", "bydey"),
+                ("fricker22", "benteezy"),
+                ("thisisnotyoho", "vortexofdoom"),
+                ("shkoople", "caznode"),
+                ("chexhuman", "lanxion"),
+                ("bambooshadow", "relkin"),
+            ]
+            .into_iter()
+            .map(From::from)
+            .collect(),
+            vec![
+                ("relkin", "benteezy"),
+                ("buane", "lanxion"),
+                ("caznode", "bydey"),
+                ("mooglemod", "vortexofdoom"),
+                ("doomtap", "shkoople"),
+                ("eriror", "thisisnotyoho"),
+                ("fricker22", "bambooshadow"),
+                ("foxlisk", "chexhuman"),
+            ]
+            .into_iter()
+            .map(From::from)
+            .collect(),
+            vec![
+                ("relkin", "thisisnotyoho"),
+                ("caznode", "mooglemod"),
+                ("buane", "bambooshadow"),
+                ("shkoople", "chexhuman"),
+                ("bydey", "vortexofdoom"),
+                ("benteezy", "lanxion"),
+                ("doomtap", "fricker22"),
+                ("eriror", "foxlisk"),
+            ]
+            .into_iter()
+            .map(From::from)
+            .collect(),
+            vec![
+                ("benteezy", "thisisnotyoho"),
+                ("bydey", "bambooshadow"),
+                ("chexhuman", "mooglemod"),
+                ("lanxion", "vortexofdoom"),
+                ("buane", "fricker22"),
+                ("shkoople", "relkin"),
+                ("foxlisk", "caznode"),
+                ("eriror", "doomtap"),
+            ]
+            .into_iter()
+            .map(From::from)
+            .collect(),
```

Comment:
it would probably be nicer to write this as like an `.into_iter().map(|i| i.into_iter().map(From::from).collect()).collect()` on the outer vec